Lint is failing
```
/home/travis/build/LLK/scratch-vm/src/blocks/scratch3_pen.js
  324:1   error  Expected indentation of 24 spaces but found 20  indent
  415:13  error  'rgb' is never reassigned. Use 'const' instead  prefer-const
  436:1   error  Expected indentation of 8 spaces but found 12   indent
  437:1   error  Expected indentation of 12 spaces but found 16  indent
  438:1   error  Expected indentation of 12 spaces but found 16  indent
  439:1   error  Expected indentation of 8 spaces but found 12   indent
  440:1   error  Expected indentation of 12 spaces but found 16  indent
  441:1   error  Expected indentation of 12 spaces but found 16  indent
  442:1   error  Expected indentation of 8 spaces but found 12   indent
  443:1   error  Expected indentation of 12 spaces but found 16  indent
  444:1   error  Expected indentation of 12 spaces but found 16  indent
  445:1   error  Expected indentation of 8 spaces but found 12   indent
  446:1   error  Expected indentation of 12 spaces but found 16  indent
  447:1   error  Expected indentation of 12 spaces but found 16  indent
  448:1   error  Expected indentation of 8 spaces but found 12   indent
  449:1   error  Expected indentation of 12 spaces but found 16  indent
```

--- a/src/blocks/scratch3_pen.js
+++ b/src/blocks/scratch3_pen.js
@@ -321,7 +321,7 @@ class Scratch3PenBlocks {
             menus: {
                 colorParam:
                     [ColorParam.COLOR, ColorParam.SATURATION,
-                    ColorParam.BRIGHTNESS, ColorParam.TRANSPARENCY]
+                        ColorParam.BRIGHTNESS, ColorParam.TRANSPARENCY]
             }
         };
     }
@@ -412,7 +412,7 @@ class Scratch3PenBlocks {
      * @private
      */
     _updatePenColor (penState) {
-        let rgb = Color.hsvToRgb({
+        const rgb = Color.hsvToRgb({
             h: penState.color * 360 / 100,
             s: penState.saturation / 100,
             v: penState.brightness / 100
@@ -433,21 +433,20 @@ class Scratch3PenBlocks {
      */
     _setOrChangeColorParam (param, value, penState, change) {
         switch (param) {
-            case ColorParam.COLOR:
-                penState.color = this._wrapColor(value + (change ? penState.color : 0));
-                break;
-            case ColorParam.SATURATION:
-                penState.saturation = this._clampColorParam(value + (change ? penState.saturation : 0));
-                break;
-            case ColorParam.BRIGHTNESS:
-                penState.brightness = this._clampColorParam(value + (change ? penState.brightness : 0));
-                break;
-            case ColorParam.TRANSPARENCY:
-                penState.transparency = this._clampColorParam(value + (change ? penState.transparency : 0));
-                break;
-            default:
-                log.warn(`Tried to set or change unknown color parameter: ${param}`);
-
+        case ColorParam.COLOR:
+            penState.color = this._wrapColor(value + (change ? penState.color : 0));
+            break;
+        case ColorParam.SATURATION:
+            penState.saturation = this._clampColorParam(value + (change ? penState.saturation : 0));
+            break;
+        case ColorParam.BRIGHTNESS:
+            penState.brightness = this._clampColorParam(value + (change ? penState.brightness : 0));
+            break;
+        case ColorParam.TRANSPARENCY:
+            penState.transparency = this._clampColorParam(value + (change ? penState.transparency : 0));
+            break;
+        default:
+            log.warn(`Tried to set or change unknown color parameter: ${param}`);
         }
         this._updatePenColor(penState);
     }

